lib/ScreenSizeController.ts

```diff
@@ -1,40 +1,59 @@
 import type { ReactiveControllerHost, ReactiveController } from 'lit';
 
-import { bound } from '@patternfly/pfe-core/decorators/bound.js';
-
 import {
-  desktopLargeBreakpoint,
-  desktopSmallBreakpoint,
-  tabletLandscapeBreakpoint,
-  tabletPortraitBreakpoint,
-  mobileLandscapeBreakpoint,
-  mobilePortraitBreakpoint,
-} from './tokens.js';
-
-export type BreakpointKey =
-  | 'mobile'
-  | 'mobileXl'
-  | 'desktopLarge'
-  | 'desktopSmall'
-  | 'tabletLandscape'
-  | 'tabletPortrait'
-  | 'mobileLandscape'
-  | 'mobilePortrait'
+  Breakpoint2xsMax,
+  Media2xl,
+  MediaLg,
+  MediaMd,
+  MediaSm,
+  MediaXl,
+  MediaXs,
+} from '@rhds/tokens/media.js';
+
+type BreakpointKey =
+  | '2xs'
```

Comment:
Should this key be `2xsMax` as it references a `max-width` value `Breakpoint2xsMax`.  If we added the `2xs` min-width value later?

```suggestion
  | '2xsMax'
```

Reply:
Let's keep it as is for now. It's the edge case, and if the tokens change we'll have to redo this anyways